projects/project-player-library/src/lib/pages/details-page/details-page.component.html

```diff
@@ -1 +1,68 @@
-<p>Project details-page</p>
+<mat-card class="details-card">
+    <span class="details-card-header">
+        <h4>{{projectDetails.title}}</h4>
+    </span>
+    <mat-card-content>
+      <div style="display: flex;">
+          <lib-icon-list [items]="projectActions" [submittedImprovement]="submitted" (newItemEvent)="iconListAction($event)"></lib-icon-list>
+      </div>
+        <mat-card class="status-bar">
+            <mat-card-content  *ngIf="!submitted">
+                <h4>{{"PROGRESS" | translate}}</h4>
+                <p>{{completedCount}}/{{projectDetails.tasks.length}} {{"TASK_COMPLETED" | translate}}</p>
+                <mat-progress-bar class="progress-bar"   mode="determinate" [value]="progressValue"></mat-progress-bar>
+            </mat-card-content>
+            <mat-card-content class="after-submission"  *ngIf="submitted">
+                <span class="material-symbols-outlined">
+                    <mat-icon aria-hidden="false"  aria-label="Example verified icon" fontIcon="verified"></mat-icon>
+
+                    </span>
+                <h3>{{"PROGRESS_COMPLETE" | translate}}</h3>
+            </mat-card-content>
+        </mat-card>
+        <div *ngIf="progressValue === 100 && !submitted" class="progress-alert">
+            <h4>{{"PROGRESS_ALERT"| translate}} <span> <mat-icon  fontIcon="arrow_circle_down"></mat-icon> </span>
+            </h4>
+
+        </div>
+    </mat-card-content>
+    <mat-card-actions *ngIf="!submitted" class="btns">
+        <button  mat-flat-button [disabled]="progressValue < 100" (click)="submitImprovement()"   color="accent">{{ "SUBMIT_IMPROVEMENT" | translate}}</button>
```

Comment:
Will the button be hidden when the project is submitted?

Reply:
yes